### PR TITLE
Raise minimum JDK version to 17

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <github@klug.nrw>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -327,8 +328,12 @@ Import-Package: \\
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>3.0.0-M5</version>
+          <version>3.0.0-M6</version>
```
Comment (wborn) [inline pom.xml:331]:
```suggestion
          <version>3.0.0-M7</version>
```

**Review by wborn — CHANGES_REQUESTED** [review]
Java 11 should also be removed from the matrix.

https://github.com/openhab/openhab-webui/blob/e39ce7f6bf1c5dce99eec7128db6744d04ddc4d0/.github/workflows/ci-build.yml#L21

**Review by wborn — CHANGES_REQUESTED** [review]
And updated in those other lines:

https://github.com/openhab/openhab-webui/blob/e39ce7f6bf1c5dce99eec7128db6744d04ddc4d0/.github/workflows/ci-build.yml#L59

**Review by wborn — CHANGES_REQUESTED** [review]
Can you also update [the other one in ci-build.yml](https://github.com/J-N-K/openhab-webui/blob/b24b41f88c78eb76d3b9f6fafbd32c8ae99b5fa2/.github/workflows/ci-build.yml#L59)?

**Review by wborn — APPROVED** [review]
LGTM now! WDYT @openhab/webui-maintainers?

**Review by kaikreuzer — APPROVED** [review]

**Pull request merged**